geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ListGatewayCommand.java

```diff
@@ -149,8 +149,12 @@ protected void accumulateListGatewayResult(CompositeResultData crd,
         gatewayReceiverData.accumulate(CliStrings.RESULT_PORT, entry.getValue().getPort());
         gatewayReceiverData.accumulate(CliStrings.RESULT_SENDERS_COUNT,
             entry.getValue().getClientConnectionCount());
-        gatewayReceiverData.accumulate(CliStrings.RESULT_SENDER_CONNECTED,
-            Arrays.stream(entry.getValue().getConnectedGatewaySenders()).collect(joining(", ")));
+        if (entry.getValue() == null || entry.getValue().getConnectedGatewaySenders() == null) {
+          gatewayReceiverData.accumulate(CliStrings.RESULT_SENDER_CONNECTED, "");
+        } else {
+          gatewayReceiverData.accumulate(CliStrings.RESULT_SENDER_CONNECTED,
+              Arrays.stream(entry.getValue().getConnectedGatewaySenders()).collect(joining(", ")));
+        }
```

Comment:
👍 I am okay adding a null check, but `getConnectedGatewaySenders()` never returns a null.

Reply:
I don't know how this happens either. the stack trace shows that getConnectGatewaySenders() does return null, and it definitely does not happen in the test body. The error happened after the test is completed, but scrapping the log for suspect strings.